hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/infra/StorageAccessTracker.java

```diff
@@ -32,19 +34,20 @@
  * Tracks storage accesses for a particular transaction.
  */
 public class StorageAccessTracker {
-    private static final Function<Long, Map<UInt256, StorageAccess>> MAP_FACTORY = ignored -> new TreeMap<>();
-    private final Map<Long, Map<UInt256, StorageAccess>> accessesByContract = new TreeMap<>();
```

Comment:
Question: is it ok to use HashMap instead of TreeMap here? I made the change because TreeMap wants a Comparable key and ContractID is not Comparable.

Reply:
Fixed